sdk/client/lusidTools.ts

```diff
@@ -0,0 +1,117 @@
+// Test specific libraries
```

Comment:
Let's reserve the name lusidTools for another package (separate GitHub repo) which will sit on top of the SDKs. Perhaps we can call this utilities or similar.